coderd/database/migrations/000063_resource_type_group.up.sql

```diff
@@ -1,5 +1 @@
-BEGIN;
-
-ALTER TYPE resource_type ADD VALUE IF NOT EXISTS 'group'; 
-
-COMMIT;
+ALTER TYPE resource_type ADD VALUE IF NOT EXISTS 'group';
```

Comment:
Should we also do the new table + rename dance here as well?
Seems safer to just always do it that way instead of someone suddenly adding a new migration referencing this enum type and wondering why their new migration doesn't work.

Reply:
I only went back historically and fixed usage of `ADD VALUE` if we reference the newly added value later in a different migration. It is otherwise fine to use within a transaction.

Reply:
I'll fix all of them in a follow up :+1: